src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -2800,6 +2927,28 @@ module Mutations = struct
     | `Bootstrapping ->
         return (Error "Daemon is bootstrapping")
 
+  let send_snapp_command coda parties =
+    match Mina_commands.setup_and_submit_snapp_command coda parties with
+    | `Active f -> (
+        match%map f with
+        | Ok parties ->
+            let cmd =
+              { Types.SnappCommand.With_status.data = parties
+              ; status = Unknown
```

Comment:
Should we add another status for this? `Enqueued` or something to show that the transaction was successfully added to the pool?

Reply:
I will do this in #9469, where I factored out `Status` from `User_command` and `Snapp_command`.

Reply:
Done in #9469, and made the comparable change for user commands.